packages/components/src/ProgressBar/ProgressBar.tsx

```diff
@@ -6,20 +6,43 @@ import { Label } from "./subcomponents/Label"
 import { calculatePercentage } from "./utils/calculatePercentage"
 import styles from "./ProgressBar.module.scss"
 
-export type ProgressBarProps = {
+export type PogressBarMood = {
+  /**
+   * @deprecated Use `color` prop instead
+   */
+  mood: "positive" | "informative" | "negative" | "cautionary"
+  color?: never
+}
+
+export type PogressBarColor = {
+  /**
+   * @deprecated Use `color` prop instead
+   */
+  mood?: never
+  /**
+   * If transitioning from `mood`:
+   * - `cautionary` -> `Yellow`
+   * - `informative` -> `Blue`
+   * - `negative` -> `Red`
+   * - `positive` -> `Green`
```

Comment:
To match the actual string literals
```suggestion
   * - `cautionary` -> `yellow`
   * - `informative` -> `blue`
   * - `negative` -> `red`
   * - `positive` -> `green`
```